tools/file_concat/main.cc

```diff
@@ -0,0 +1,65 @@
+// Copyright 2020 The Bazel Authors. All rights reserved.
```

Comment:
Have you considered using "cat" on Linux and "Add-Content" on Windows? Using standard tools may reduce maintenance  with time.

Reply:
Yes, and I came to the conclusion that I prefer having 1 tool that runs on all platforms. Having 1 script for Windows and another for Linux/macOS is (IMO) means more maintenance than having a simple custom tool. Using `cat` everywhere would be possible but undesirable because it adds a dependency on bash on Windows.